main.tex

```diff
@@ -23,6 +23,7 @@
 % Title
 \title{An Empirical Study of Invasion and Resistance
        for Iterated Prisoner's Dilemma Strategies}
+\title{Evolving Invaders and Resistors for the Iterated Prisoner's Dilemma}
```

Comment:
On reflection, I'm not sure this is the correct title. We don't spend much time discussing the training of the Invaders and Resistors...